File: code-recipes/php/test/easyship/GetHandoverSlotsRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\GetHandoverSlotsRecipe;
use Test\util\RecipeTestCase;

class GetHandoverSlotsRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new GetHandoverSlotsRecipe();
        $this->responses = [
            "easyship-listHandoverSlots"
        ];
    }
}

File: code-recipes/java/src/test/java/easyship/GetHandoverSlotsRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class GetHandoverSlotsRecipeTest extends RecipeTest {

    protected GetHandoverSlotsRecipeTest() {
        super(
                new GetHandoverSlotsRecipe(),
                List.of(
                    "easyship-listHandoverSlots"
                )
        );
    }
}
